src/datanode/src/alive_keeper.rs

```diff
@@ -380,7 +380,9 @@ impl CountdownTask {
                             }
                         },
                         Some(CountdownCommand::Reset((role, deadline))) => {
-                            let _ = self.region_server.set_region_role(self.region_id, role);
+                            if let Err(err) = self.region_server.set_region_role(self.region_id, role){
```

Comment:
Looks like the there are some issues with the format
```suggestion
                            if let Err(err) = self.region_server.set_region_role(self.region_id, role) {
```